### PR TITLE
Add a "My Settings Don't Work Anymore!" issue template

### PR DESCRIPTION
**Pull request by zadjii-msft**
Because people can't really read the pinned megathread, I don't expect them to read this either, but hopefully the "/dup" tag at the bottom will save us some triage hassle

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: "Bug report 🐛"
+name: "Bug report \U0001F41B"
```
Comment (zadjii-msft) [inline .github/ISSUE_TEMPLATE/Bug_Report.md:2]: oh no
```suggestion
name: "Bug report 🐛"
```

File: .github/ISSUE_TEMPLATE/Documentation_Issue.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-name: "Documentation Issue 📚"
+name: "Documentation Issue \U0001F4DA"
```
Comment (zadjii-msft) [inline .github/ISSUE_TEMPLATE/Documentation_Issue.md:2]:
```suggestion
name: "Documentation Issue 📚"
```

File: .github/ISSUE_TEMPLATE/Feature_Request.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-name: "Feature Request/Idea 🚀"
+name: "Feature Request/Idea \U0001F680"
```
Comment (zadjii-msft) [inline .github/ISSUE_TEMPLATE/Feature_Request.md:2]:
```suggestion
name: "Feature Request/Idea 🚀"
```

**Comment by miniksa** [comment]
I don't know how to feel about this. Is this a temporary thing for the next few weeks to stem the flow until we get past the surge?

**Comment by zadjii-msft** [comment]
> Is this a temporary thing for the next few weeks

That's what I imagined. This is 100% a showerthought PR, I'm open to discussion about it

**Comment by DHowett-MSFT** [comment]
0.11.1191 is rolling out now with the new "globals" warning. I'm gonna close this one out.

**Pull request closed without merging**